android/src/main/java/com/github/alinz/reactnativewebviewbridge/WebViewBridgePackage.java

```diff
@@ -22,9 +22,4 @@
                 new WebViewBridgeManager()
         );
     }
-
-    @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-        return Arrays.asList();
-    }
```

Comment:
This fix will only work if you also edit the build.gradle to say
`compile 'com.facebook.react:react-native:+'`
instead of
`compile 'com.facebook.react:react-native:0.19.+'`